opentech/apply/review/forms.py

```diff
@@ -78,6 +79,10 @@ def __init__(self, *args, **kwargs):
         self.submission = kwargs.pop('submission')
         super().__init__(*args, **kwargs)
 
+        if "save_draft" in self.data:
```

Comment:
Can we add this as an attribute to the form. Then reference it in the template. Removes the magic strings

Reply:
👍